### PR TITLE
Error UX changes

### PR DESCRIPTION
**Pull request by ashutshkumr**
[Redocli link](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/error-ux/artifacts/openapi.yaml&nocors)

Changes:
- Error responses have been replaced by single `default` response with `Error` as an underlying schema
- RPCs in protobuf directly encapsulate schema defined in openapi.yaml inside request or response message

### CODE REVIEW

File: artifacts/otg.proto [hunk 1]
```diff
@@ -16648,161 +16659,131 @@ message UpdateConfigRequest {
...
 message GetVersionResponse {
-  optional Version status_code_200 = 1;
-  optional ResponseError status_code_400 = 2;
-  optional ResponseError status_code_500 = 3;
+  Version version = 1;
 }
 
 
-  // Description missing in models
+// Description missing in models
```
Comment (apratimmukherjee) [inline artifacts/otg.proto:16782]: Not sure why this Description missing is still coming.

File: artifacts/otg.proto [hunk 2]
```diff
@@ -16648,161 +16659,131 @@ message UpdateConfigRequest {
...
+//
+// For all RPCs defined in this service, API Server SHOULD provide JSON
+// representation of `Error` message as an error string upon failure, ensuring
+// name of enum constants (instead of value) is present in the representation
```
Comment (apratimmukherjee) [inline artifacts/otg.proto:16786]: ...(instead of value) is present in the representation --> maybe insert '.. is present for the 'kind' attribute ...

**Review by apratimmukherjee — APPROVED** [review]
Two minor documentation nits. Rest looks okay.

File: artifacts/otg.proto (hunk 2 above)
Comment (ashutshkumr) [reply artifacts/otg.proto:16786]: done

File: artifacts/otg.proto (hunk 1 above)
Comment (ashutshkumr) [reply artifacts/otg.proto:16782]: known issue, will be resolved as a separate PR

**Pull request merged**